### PR TITLE
UI and routing

### PR DESCRIPTION
**Pull request by MaruthiPothuganti**
-  Basic  Layout with Dummy Components.
- HomePage, Explore, Playlists, Liked Videos and Watch Later Pages Done.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="overflow-y: hidden;">
```
Comment (TejasShekar) [inline public/index.html]: Inline styles ❌

**Review by ashishkatiyar2404 — APPROVED** [review]
Functionality looks good to me. Good work maruthi.
You can merge now and work on other functionality of the project

File: src/Components/SideNav.jsx [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
+    <Box
+      sx={{
+        width: "200px",
+        display: { xs: "none", md: "flex" },
+      }}
+      role="presentation"
+      borderRight={1}
+      borderColor="#f3e5f5"
+    >
+      <List
```
Comment (TejasShekar) [inline src/Components/SideNav.jsx]: List component seems to be repeating. Extracting it as a seperate component will keep code clean

**Review by TejasShekar — APPROVED** [review]
Have added a two suggestions. Rest looks good to merge.

File: src/Components/SideNav.jsx (hunk 2 above)
Comment (MaruthiPothuganti) [reply src/Components/SideNav.jsx]: Yep. Thankyou bro. Extracted it as a component.

**Pull request merged**